localization/strings/en-GB/Resources.resw

```diff
@@ -1100,7 +1100,7 @@ Error code: {}</value>
     <value>Failed to query networking mode</value>
   </data>
   <data name="MessageMismatchedKernelModulesError" xml:space="preserve">
-    <value>Customised kernel modules were supplied without a specifying a customised kernel. Please see https://aka.ms/wslcustomkernel for more information.</value>
+    <value>Customised kernel modules were supplied without specifying a customised kernel. Please see https://aka.ms/wslcustomkernel for more information.</value>
```

Comment:
This file is generated, but I think this is fine because you also updated the en-us translation which will result in this one being updated anyway. Fine to leave as-is.